wagtail/admin/widgets/tags.py

```diff
@@ -31,6 +31,13 @@ def get_context(self, name, value, attrs):
         else:
             free_tagging = self.free_tagging
 
+        tag_spaces_allowed = getattr(settings, "TAG_SPACES_ALLOWED", True)
+        if tag_spaces_allowed:
+            tag_help_text = "Multi-word tags with space will be enclosed in quotes and separated by comma."
+        else:
+            tag_help_text = "Tags can only consist of a single word, no spaces allowed."
+
+        context["widget"]["tag_help_text"] = tag_help_text
```

Comment:
as noted above - this should be `help_text` as it is simpler.
```suggestion
        context["widget"]["help_text"] = tag_help_text
```